src/components/SideNav.tsx

```diff
@@ -163,7 +162,7 @@ const SideNav: React.FC<IProps> = ({ path }) => {
       borderRight="1px solid"
       borderRightColor="border"
       display={{ base: "none", lg: "block" }}
-      aria-label={t("nav-developers-docs")}
+      aria-label={t("common:nav-developers-docs")}
```

Comment:
>       aria-label={t("common:nav-developers-docs")}

@nhsz Do we need this `common:` prefix here?

Reply:
I see this in a few places, so assuming so... do we need to declare that for any of the "common" strings within a component?

Reply:
Prefix just needed when other namespaces are being loaded in the same component

Reply:
Got it! Thanks!